### PR TITLE
CNV-5431 Automatic renewal of TLS certificates

### PR DESCRIPTION
**Pull request by lmandavi**
This PR is associated with https://issues.redhat.com/browse/CNV-3541. It is currently still a placeholder until the content is drafted.

### CODE REVIEW

**Comment by lmandavi** [comment]
@rmohr Requesting review of the content:
https://cnv-3541-cert-renewal--ocpdocs.netlify.app/openshift-enterprise/latest/virt/node_maintenance/virt-certificate-renewal.html

File: modules/virt-automatic-certificate-renewal.adoc [hunk 1]
```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * virt/node_maintenance/virt-certificate-renewal.adoc
+
+[id="virt-automatic-certificate-renewal_{context}"]
+= Automatic renewal of TLS certificates
+
+TLS certificates for {VirtProductName} are created during installation for these {VirtProductName} components:
+
+ * KubeVirt API
```
Comment (rmohr) [inline modules/virt-automatic-certificate-renewal.adoc:10]: Not just the API rotates. I would just write `KubeVirt`.

**Review by rmohr — COMMENTED** [review]
One comment and it looks like we leave the manual rotation code in place. The manual rotation will as we speak not work properly for 2.4 if one follows the 2.3 guideline on it.

**Comment by lmandavi** [comment]
> One comment and it looks like we leave the manual rotation code in place. The manual rotation will as we speak not work properly for 2.4 if one follows the 2.3 guideline on it.

@rmohr My thought is to delete the content piece on manual refresh of TLS certificates for this release. Checking with my team.

**Comment by lmandavi** [comment]
This PR is not relevant and it is outdated. The work associated with this Jira story was completed via another PR. Closing this PR.

**Pull request closed without merging**